data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1853 @@
+			label end
+			`	"Attempting to pass as ranchers, the members are. On a trip to <planet>, but an argument with their original transport, they had. If stopped, you are, tell them that, you should. Remember, <date>. Hopefully still there when you arrive, all <bunks> of them are." He wishes you good luck, and you're brought out of the ship, with the Saryds that found you staying inside. Not long after you've left it, you see the Spire take off and leave the planet.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort with the <bunks> bunks for the ranchers hasn't arrived in the system yet. Better depart and wait for it.`
+
+mission "Resistance: Evacuation 4"
+	name "Transport The Ranchers"
+	description "Bring the <bunks> ranchers to their world, <destination>."
+	passengers 47
+	landing
+	source "Chosen Nexus"
+	destination "Shifting Sand"
+	to offer
+		has "Resistance: Evacuation 3: done"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch License: done"
+	to fail
+		has "Heliarch Investigation 2: active"
+	on offer
+		conversation
+			`The ranchers arrive to the hangars shortly after you land, most of them Arachi, with a few Saryds and Kimek in their midst. They come into your ship, all <bunks> of them present.`
+			branch resistance
+				has "Resistance: Questions: done"
+			`	"All ready to leave, we are," one Arach tells you as most of the others head to find their bunks.`
+			choice
+				`	"Got it, we'll leave soon."`
+					goto accept
+				`	"How did getting that loan go?"`
+			`	"What?" She looks at you with puzzled eyes, until another Arach lightly whacks one of her legs. "Ah, yes, the loan. Well enough. Agreed to grant us the amount needed, they have." She looks back at the other Arach, who stares at her for a moment. "Well, very tired we all are, Captain, so head to my bunk, I will."`
+				goto accept
+			label resistance
+			`	"Thought I had, that left behind we would be," one Arach tells you after you've closed the hatch. "Hurry to <planet>, you should. Risk an 'inspection' by the Heliarchs, we cannot."`
+			choice
+				`	"Got it, we'll leave soon."`
+					goto accept
+				`	"Is everyone here?"`
+			`	"Thankfully, yes. Lost nobody this time, we have," she says. "Less brutish on their searches on worlds like <origin>, the Heliarchs are. Managed to avoid them well enough, we did. Or, at least, hope so I do."`
+			choice
+				`	"Well I'm glad everyone's safe. We'll leave soon."`
+					goto accept
+				`	"What do you mean? What's different about worlds like this one?"`
+			`	"What? Obvious is it not? In Saryd territory, we are. Come on now, ready to leave, we are."`
+			label accept
+			`	They all move to their bunks, and you set a course for <destination>.`
```

Comment:
```suggestion
			`	They all move to their quarters, and you set a course for <destination>.`
```
I would think there would be a little more luxury for passengers than is given to soldiers. This might depend on your ship more, though.

Reply:
I believe bunks is the standard term used throughout the game, don't recall any usage of quarters.

Reply:
OK, it's probably fine.